src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -7,6 +7,12 @@ import com.typesafe.scalalogging.LazyLogging
 import scala.concurrent.duration.Duration
 import scala.util.{Success, Failure, Try}
 
+case class AggregatedException[A](exceptions: Seq[Failure[A]]) extends Exception {
+  override def getMessage: String = {
+    exceptions.map(_.exception.getMessage).mkString("\n")
+  }
+}
```

Comment:
Indeed, though there's some nuance that would be good to make sure a third party library supports... for example, here we're just doing `.getMessage` and concatenating them all together with `\n`, but what if we want to print stack traces for each exception?  Or distinguish the messages based on the type of exception?